docs/build/qiskit-qasm2.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Interoperate Qiskit with OpenQASM 2
+description: Convert code between OpenQASM 2 and Qiskit
+
+---
+
+# Interoperate Qiskit with OpenQASM 2
+
+Qiskit provides some tools for converting between OpenQASM representations of quantum programs, and the QuantumCircuit class.
+
+<Admonition type="note">
+This function is still in the exploratory phase.  Therefore, it is likely that the syntax and capabilities will evolve.
+</Admonition> 
+
+## Import an OpenQASM 2 program into Qiskit
+
+Currently two high-level functions are available for importing from OpenQASM 2 into Qiskit. These functions are `load()`, which takes a file name, and `loads()`, which takes the program itself as a string. 
+
+```python
+qiskit.qasm2.load(filename, *, include_path=('.',), include_input_directory='append', custom_instructions=(), custom_classical=(), strict=False)
+```
+
+Parameters:
+  - `filename` (str | PathLike) – The path to the OpenQASM 2 program in an ASCII or UTF-8 string.
+  - `include_path` (Iterable[str | PathLike]) – Order of directories to search when evluating include statements. The directories are tried from index 0 onwards and the first match is used. The import `qelib1.inc` is special; it is always found before looking in the include path, and contains exactly the content of the [paper describing the OpenQASM 2 language](https://arxiv.org/abs/1707.03429). The gates in this include file are mapped to circuit-library gate objects defined by Qiskit.
+  - `include_input_directory` (Literal[`append`, `prepend`] | None) – Whether to add the directory of the input file to the include_path, and if so, whether to append it to search last, or prepend it to search first. Pass `None` to suppress adding this directory entirely.
+  - `custom_instructions` (Iterable[CustomInstruction]) – Any custom constructors that should be used for specific gates or opaque instructions during circuit construction. See [Specifying custom instructions](#custom-instructions) for details.
+  - `custom_classical` (Iterable[CustomClassical]) – Any custom classical functions that should be used during the parsing of classical expressions. See [Specifying custom classical functions](#custom-classical) for details.
+  - `strict` (bool) – whether to run in [strict mode](#strict). 
```

Comment:
Seems reasonable to me. Are there examples we need to include that we don't already have? (In case you're surprised to see that this PR has been merged, note that the issue for continuing this work is https://github.com/Qiskit/documentation/issues/197)